packages/swingset-liveslots/package.json

```
@@ -38,6 +41,7 @@
   "files": [
     "src/**/*.js",
     "src/**/*.d.ts",
+    "src/**/*.map",
```

We should be able to change this to 

```suggestion
    "src",
```